addons/zylann.editor_debugger/dock.gd

```diff
@@ -295,16 +289,18 @@ func _pick(root: Node, mpos: Vector2, level := 0) -> Node:
 	
 	for i in root.get_child_count(true):
 		var child := root.get_child(i, true)
+		var child_canvas_item := child as CanvasItem
+		var child_control := child as Control
 		
-		if (child is CanvasItem and not child.visible):
+		if (child_canvas_item and not child_canvas_item.visible):
```

Comment:
Same here, add `!= null`. Also, prefer moving `var child_control` lower, closer to where it actually starts being used.